portfolio/src/main/webapp/index.html

```diff
@@ -5,18 +5,13 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-    <style>
-        body {
-            background-image: url('https://i.gifer.com/GHSe.gif');
-            }
-    </style>
     <link href="https://fonts.googleapis.com/css?family=Annie+Use+Your+Telescope|Nanum+Pen+Script&display=swap" rel="stylesheet">
   </head>
   <body>
       <div class="nav">
   <a class="active" href="#home">Home</a>
-  <a href="#news">News</a>
-  <a href="#contact">Contact</a>
+  <!-- <a href="#news">News</a> -->
```

Comment:
We generally prefer not to submit commented out code. I'd recommend you go ahead and just delete this line for now.